flocker/node/agents/functional/test_ebs.py

```diff
@@ -315,11 +312,19 @@ class EBSProfiledBlockDeviceAPIInterfaceTests(
     pass
 
 
+class VolumeStub(object):
```

Comment:
You can create a ``Volume`` with an ``id`` before calling out to AWS, but you can't instantiate with any other attributes.